test/sql/testdata.sql

```diff
@@ -1620,23 +1620,27 @@ INSERT INTO sagseventinfo (
-INSERT INTO koordinat (registreringfra,sagseventfraid,sridid,x,y,z,sx,sy,sz,t,transformeret,fejlmeldt,punktid) VALUES (to_timestamp_tz('2022-01-18 12:00:07','YYYY-MM-DD HH24:MI:SS'),'sagevent-aaaa-bbbb-0008-000000000003',1,3501320.66121,629366.08586,5276291.21018,5,5,15,to_timestamp_tz('2015-07-28 12:00:00','YYYY-MM-DD HH24:MI:SS'),'false','false','301b8578-8cc8-48a8-8446-541f31482f86');
-INSERT INTO koordinat (registreringfra,sagseventfraid,sridid,x,y,z,sx,sy,sz,t,transformeret,fejlmeldt,punktid) VALUES (to_timestamp_tz('2022-01-18 12:00:08','YYYY-MM-DD HH24:MI:SS'),'sagevent-aaaa-bbbb-0008-000000000003',1,3501320.66013,629366.08805,5276291.21349,5,5,15,to_timestamp_tz('2015-09-22 12:00:00','YYYY-MM-DD HH24:MI:SS'),'false','false','301b8578-8cc8-48a8-8446-541f31482f86');
-INSERT INTO koordinat (registreringfra,sagseventfraid,sridid,x,y,z,sx,sy,sz,t,transformeret,fejlmeldt,punktid) VALUES (to_timestamp_tz('2022-01-18 12:00:09','YYYY-MM-DD HH24:MI:SS'),'sagevent-aaaa-bbbb-0008-000000000003',1,3501320.65413,629366.09336,5276291.21746,5,5,15,to_timestamp_tz('2016-02-23 12:00:00','YYYY-MM-DD HH24:MI:SS'),'false','false','301b8578-8cc8-48a8-8446-541f31482f86');
-INSERT INTO koordinat (registreringfra,sagseventfraid,sridid,x,y,z,sx,sy,sz,t,transformeret,fejlmeldt,punktid) VALUES (to_timestamp_tz('2022-01-18 12:00:10','YYYY-MM-DD HH24:MI:SS'),'sagevent-aaaa-bbbb-0008-000000000003',1,3501291.06444,629308.42028,5276318.75039,5,5,15,to_timestamp_tz('2016-02-23 12:00:00','YYYY-MM-DD HH24:MI:SS'),'false','false','4b4c5c17-32e8-495d-a598-cdf42e0892de');
-INSERT INTO koordinat (registreringfra,sagseventfraid,sridid,x,y,z,sx,sy,sz,t,transformeret,fejlmeldt,punktid) VALUES (to_timestamp_tz('2022-01-18 12:00:11','YYYY-MM-DD HH24:MI:SS'),'sagevent-aaaa-bbbb-0008-000000000003',2,3501291.02891,629308.45367,5276318.77000,5,5,15,to_timestamp_tz('2018-06-05 12:00:00','YYYY-MM-DD HH24:MI:SS'),'false','false','4b4c5c17-32e8-495d-a598-cdf42e0892de');
-INSERT INTO koordinat (registreringfra,sagseventfraid,sridid,x,y,z,sx,sy,sz,t,transformeret,fejlmeldt,punktid) VALUES (to_timestamp_tz('2022-01-18 12:00:12','YYYY-MM-DD HH24:MI:SS'),'sagevent-aaaa-bbbb-0008-000000000003',2,3501291.02211,629308.45644,5276318.76708,5,5,15,to_timestamp_tz('2018-09-25 12:00:00','YYYY-MM-DD HH24:MI:SS'),'false','false','4b4c5c17-32e8-495d-a598-cdf42e0892de');
+INSERT INTO koordinat (registreringfra,sagseventfraid,sridid,x,y,z,sx,sy,sz,t,transformeret,fejlmeldt,punktid) VALUES (to_timestamp_tz('2022-01-18 12:00:00','YYYY-MM-DD HH24:MI:SS'),'sagevent-aaaa-bbbb-0008-000000000003',1,3501320.82959,629365.92108,5276291.10823,5,5,15,to_timestamp_tz('2004-06-01 00:00:00','YYYY-MM-DD HH24:MI:SS'),'false','false','301b8578-8cc8-48a8-8446-541f31482f86');
```

Comment:
Idé: Samme sagsevent-IDer gentages mange gange, hvilket kan forstyrrer læsningen lidt, og det kan være svært at se (og huske), hvilket ID, der bruges hvor. Hvis der ikke er for mange IDer, kan vi så ikke bruge variable til at sætte disse én gang for alle?

Reply:
Jeg aner ikke hvordan man gør det, eller om det overhovedet er muligt. Jeg tror det ikke. Variabler og SQL er et bøvlet felt. Her besværliggøres det helt enormt af at alle udtryk i denne fil afvikles som selvstændige operationer, så det er ikke bare lige at lave en variabel i starten af filen og så er den ude af verden. Uanset hvad tænker jeg det er udenfor scope, men hvis du har en lys ide til forbedring er det værd at se på på et senere tidspunkt

Reply:
Ok, også med evt. senere tilpasning. Overvejelsen kom, fordi rettelsen potentielt kunne gøre hele filen mere klar at læse igennem. More research is needed.